packages/web/src/components/Resource.tsx

```diff
@@ -127,11 +129,11 @@ export const Resource = () => {
         {resource.phone && (
           <ListItem component="div">
             <ListItemIcon classes={listIconClasses}>
-              <PhoneIcon />
+              <PhoneIcon titleAccess={t("phoneNumber")} />
             </ListItemIcon>
             <ListItemText>
               <Typography component="h2" variant="srOnly">
-                Phone
+                {t("phoneNumber")}
```

Comment:
I like your thinking here, but in this case I don't think we need to have the title on the icon. The `srOnly` variant of the following heading sets this as hidden text that is available to screen readers.

Adding a title to the icon as well means a SR would read "phone number" twice.

Reply:
Good point! f4948d9cda9b3d0a5c638a5961ee256ef0cf7ed2